### DISCUSSION TITLE
Reveal repetitive deprecation warnings from sass-loader

### DISCUSSION THREAD

**Discussion by thany** [webpack/webpack#16641] [General]
Currently our project, even though it builds just fine, shows a warning during build:

```
WARNING in ./src/styles/styles.scss (./node_modules/css-loader/dist/cjs.js??clonedRuleSet-3[0].rules[0].use[1]!./node_modules/sass-loader/dist/cjs.js??clonedRuleSet-3[0].rules[0].use[2]!./src/styles/styles.scss)
Module Warning (from ./node_modules/sass-loader/dist/cjs.js):
4 repetitive deprecation warnings omitted.

null
 @ ./src/styles/styles.scss 4:14-197
 @ ./src/main.js 27:0-30
```

It doesn't show *what* is repeated 4 times, just that something is.

I've tried with the `--stats-error-details` flag, but that just shows the same repetition message, only with a stack trace that is not useful for me.

I can't seem to figure how to disable this repetition warning, and show the actual 4 warnings themselves. How can I do that?

**Comment by alexander-akait** [comment]
Hello, wierd, maybe bug in `sass`, can you create reproducible test repo, thank you

**Reply by thany** [reply]
That's really difficult unfortunately. This is not a project I created, and it's very old and very complicated. I was just hoping for a switch to turn off the repetition message and "expand" it into the actual warnings...

**Reply by alexander-akait** [reply]
Can you try to debug this line - https://github.com/webpack-contrib/sass-loader/blob/master/src/index.js#L68
